<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0A0811;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>741</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/TuongDii.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>14</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Good to see you</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Arial&quot;;
color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Tuong Dii</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>701</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #333135;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/icons8-search-30.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>12</y>
        <width>641</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>674</x>
       <y>14</y>
       <width>49</width>
       <height>49</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/icons8-qr-code-64.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>18</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/icons8-home-30.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>49</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/icons8-images-folder-48.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/icons8-friends-48.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>563</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/icons8-qr-64.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: #FFFFFF;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>178</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;
font: 700 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: #24222A;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>177</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
font: 700 10pt &quot;Arial&quot;;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Manager</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>170</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: #24222A;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>178</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
font: 700 10pt &quot;Arial&quot;;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Coding</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>170</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: #24222A;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>178</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
font: 700 10pt &quot;Arial&quot;;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Media</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>R&amp;D Manager</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewImg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/anhPhuoc.png</normaloff>img/anhPhuoc.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>91</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>17</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Arial&quot;;
color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Phuoc Ding</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>R&amp;D Media</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewImg3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>7</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/bao (2).png</normaloff>img/bao (2).png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>91</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view3">
     <property name="geometry">
      <rect>
       <x>104</x>
       <y>16</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Arial&quot;;
color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Gia Bao</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_26">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>R&amp;D Coding</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewImg4">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>7</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/khoa.png</normaloff>img/khoa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>91</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view4">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>15</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Arial&quot;;
color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Khoa Dao</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_23">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>R&amp;D Coding</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view1">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>14</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Arial&quot;;
color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Tuong Dii</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewImg1">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>6</y>
       <width>101</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/TuongDii.png</normaloff>img/TuongDii.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>91</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_32">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>R&amp;D Coding</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewImg5">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>6</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/Minh.png</normaloff>img/Minh.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>91</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view5">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>18</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Arial&quot;;
color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Nguyen Min</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_8">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #24222A;
border-radius: 15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_29">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 10pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>R&amp;D Media</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_viewImg2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/chiHan.png</normaloff>img/chiHan.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>91</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>11</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 12pt &quot;Arial&quot;;
color: #ffffff;
</string>
     </property>
     <property name="text">
      <string>Ngoc Han</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
